### PR TITLE
feat(cron): support for 'pdo' outputFormat

### PR DESCRIPTION
**Pull request by tamil-rss**
#24

### CODE REVIEW

**Comment by tamil-rss** [comment]
There is a deviation in the **constructor** method for `MetaDataStorageHandlerPdo.php` between SimpleSAMLphp 1.x to 2.x.

### SimpleSAML 1.x

```php
public function __construct($config)
    {
        assert(is_array($config));

        $this->db = \SimpleSAML\Database::getInstance();
    }
``` 

### SimpleSAML 2.x
```php
public function __construct(
        /** @scrutinizer ignore-unused */ Configuration $globalConfig,
        /** @scrutinizer ignore-unused */ array $config,
    ) {
        parent::__construct();

        $this->db = Database::getInstance();
    }
``` 

This PR is based out of `master` branch which can support only SimpleSAML 2.x. Will create an another PR from `release-0.10` to support backward compatibility.

**Review by tvdijen — APPROVED** [review]

**Comment by tvdijen** [comment]
Don't mind the failing tests. I have yet to fix our test-suite.

**Comment by tamil-rss** [comment]
Here is the PR to support SimpleSAMLphp **1.x** - #26

**Comment by tamil-rss** [comment]
@tvdijen Can you help me merge both PRs and bump a new version in composer ?

File: src/MetaLoader.php [hunk 1]
```diff
@@ -649,16 +649,42 @@ public function writeMetadataSerialize(string $outputDir): void
...
                 $metaHandler->saveMetadata($entityId, $set, $m['metadata']);
             }
         }
     }
 
 
+    /**
+     * This function uses the `PDO` metadata handler to upsert metadata in database.
+     * 
+     * @author - Tamilselvan Sekar, University of California, Davis. <tsekar@ucdavis.edu>
```
Comment (thijskh) [inline src/MetaLoader.php:666]: Although your contribution is appreciated, please do not add @author tags. Someone may make a change to the method later etc. The git history will record who added which code when.

**Comment by tvdijen** [comment]
Hey @tamil-rss !  I have a person who wants to use this, but can you help giving a configuration example?

**Pull request merged**